CMakeLists.txt

```diff
@@ -20,11 +19,12 @@ ExternalProject_Add(glog_src
   INSTALL_COMMAND cd ../glog_src/ && make install -j 8
 )
 
-cs_add_library(${PROJECT_NAME} src/dependency_tracker.cc)
-add_dependencies(${PROJECT_NAME} glog_src)
-target_link_libraries(${PROJECT_NAME} ${CATKIN_DEVEL_PREFIX}/lib/libglog${CMAKE_SHARED_LIBRARY_SUFFIX})
-
-cs_install()
-
-cs_export(INCLUDE_DIRS ${CATKIN_DEVEL_PREFIX}/include
-  CFG_EXTRAS glog-extras.cmake)
```

Comment:
It doesn't seem to be necessary anymore. I could build the entire maplab project with these glog/gflags_catkin versions. (Only a minor change was necessary in ceres_catkin because the variable changed from `..._INCLUDE_DIR` to `..._INCLUDE_DIRS`.)

Reply:
Oh, yes that seems to be correct (looking at the content `glog-extras.cmake` for the first time ..). Unfortunately removing it (nice cleanup) is a breaking change then. Can we estimate how many packages it breaks?

Reply:
I've just done a ethz-asl wide [github search](https://github.com/search?q=org%3Aethz-asl+glog_catkin_INCLUDE_DIR&type=Code) . There seem to be three packages that copied the flawed code (also depending on glfags). Sounds manageable to me.